finality-aleph/src/testing/validator_network.rs

```diff
@@ -0,0 +1,113 @@
+use std::sync::Once;
+
+use tokio::time::Duration;
+
+use crate::testing::mocks::validator_network::scenario;
+
+static INIT: Once = Once::new();
+
+fn setup() {
+    INIT.call_once(|| {
+        env_logger::init();
```

Comment:
Maybe we could add a comment about this needing to be called only once during test running and why and that it is needed to check logs for network with `--nocapture`?

Reply:
added docs